lms/templates/courseware/proctored-exam-status.underscore

```diff
@@ -19,9 +19,14 @@
             <% } %>
         </span>
         <span class="sr timer-announce" aria-live="assertive"></span>
-        <span id="time_remaining_id">
-            <b>
-            </b>
+        <span class="exam-timer-clock">
+            <span id="time_remaining_id">
+                <b>
+                </b>
+            </span>
+            <button id="toggle_timer" title="<%- gettext("Hide Timer") %>" aria-label="<%- gettext("Hide Timer") %>">
```

Comment:
@jspayd I do not like the title attribute at all since it can only be accessed by Mouse users (by hovering).   If there is any concern that the icon will not be enough to convey the meaning of the control, then I would go with option 2, but I highly prefer option 1.